plugin.json

```diff
@@ -10,6 +10,7 @@
     "currentVersion": "1.1.11",
     "changelog": {
         "de": {
+            "1.1.12": "CON-4522: Drop shipping Einstellungen in den Import Tab verschoben",
```

Comment:
in general i would not add the changelog in this moment because you never really know with wich release your PR is realesed.
The workflow is to flag the jira ticket with the plugin version you think and then when we want to do  package test we add al changelogs for that release